library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -489,6 +501,16 @@ public Cursor query(Query query) {
         return new CursorTranslator(underlyingCursor, Downloads.Impl.DOWNLOADS_BY_BATCH_URI);
     }
 
+    public Cursor queryBatch(BatchQuery query) {
```

Comment:
I would just call it `query` the typing is enough to see you are doing a batch query